### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by avolchivschi**
setup => Installed vue-validate and changed files directory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,12 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^4.3.1",
+    "bootstrap-vue": "^2.0.0-rc.19",
     "core-js": "^2.6.5",
+    "react-bootstrap": "^0.32.4",
```
Comment (nlaptedulce) [inline package.json]: It is a library for react, remove please this row from package.js

File: package.json [hunk 2]
```diff
@@ -8,8 +8,12 @@
...
     "register-service-worker": "^1.6.2",
```
Comment (nlaptedulce) [inline package.json]: This is not necessary, it would be good to delete the package and other files related to it

**Review by aghermanHP — APPROVED** [review]
very nice worked, congratulations Arina

**Comment by avolchivschi** [comment]
very nice worked Adrian, good job))

**Review by nlaptedulce — APPROVED** [review]

**Pull request merged**